code/global.dm

```diff
@@ -5,7 +5,7 @@ var/global/list/all_areas                = list()
 var/global/list/ship_areas               = list()
 
 
-var/global/list/ships 	= list() // List of ships in the game.
+GLOBAL_LIST_EMPTY(ships) // List of ships in the game.
```

Comment:
It's a problem with these lines here, not sure exactly why it doesn't work though.